setup.py

```diff
@@ -99,6 +99,7 @@
     "flake8>=3.8.3",
     "flax>=0.3.2",
     "fugashi>=1.0",
+    "huggingface-hub>=0.0.8",
```

Comment:
Tiny detail, but I still do not understand why it's not `huggingface_hub` :)

Reply:
I think it's safer to use the dash for the deps lists or pip install, and then the underscore for the module import.

Reply:
got it, thank you